lib/assets/javascripts/cartodb/table/map/map_options_dropdown.js

```diff
@@ -124,7 +124,7 @@ cdb.admin.MapOptionsDropdown = cdb.ui.common.Dropdown.extend({
       if (!_.contains(self.collection.pluck("type"), overlay_type)) {
         self.mapOptions.set(overlay_type, false);
       }
-    })
+    });
```

Comment:
That's a good call; I was trying to clean up my linter warnings to track down a syntax error, but I'll remove these before merging.